lab_1/main.py

```diff
@@ -26,7 +30,14 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     --> ['weather', 'sunny', 'man', 'happy']
     """
     pass
+    if isinstance(tokens, list) != True or isinstance(stop_words, list) != True:
+        return []
+
+    else:
+        return [word for word in tokens if word not in stop_words]
+
 
+import collections
```

Comment:
Imports are usually placed at the top of the module.

Also, you should implement this lab without using libraries (except for `re`).